protoc-gen-kotlin/protoc-gen-kotlin-common/src/main/kotlin/pbandk/gen/Namer.kt

```diff
@@ -3,7 +3,7 @@ package pbandk.gen
 interface Namer {
     fun newTypeName(preferred: String, nameSet: Collection<String>): String
     fun newFieldName(preferred: String, nameSet: Collection<String>): String
-    fun newEnumValueName(preferred: String, nameSet: Collection<String>): String
+    fun newEnumValueClassName(typeName: String, preferred: String, nameSet: Collection<String>): String
```

Comment:
🙃 Rename to `newEnumValueTypeName`

Reply:
Bah, I should've caught this (and the rest of the renames)... will fix up shortly.